CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 ## [3.8.0] - TBD
 ### Changed
 * Updated `hotels-oss-parent` to 6.1.0 (was 5.0.0).
-* Updated `waggle-dance.sh` script to include the default log4j2 configuration.
+* Updated README with logging configuration.
```

Comment:
Doesn't need to be mentioned in the CHANGELOG as this isn't really part of a "release".

Reply:
True. Removing.